### PR TITLE
DOCS-1785 Add recommendation sub section

### PR DESCRIPTION
**Pull request by npmenard**
# Description

Sometimes SDK connections to Micro-RDK can be unstable or slow. We add a recommendation section to let the user know they can disable some defaults values to make the experience better

### CODE REVIEW

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 1]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
...
+if err != nil {
+    logger.Fatal(err)
+}
+
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:129]:
```suggestion
```
Will fix linter problem

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 2]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
 {{% card link="/build/program/" %}}
 {{< /cards >}}
 
+## Recommendations when using an SDK
+
+In some cases connection the ESP32 using an SDK can fail, be unstable or be slow, this is usually caused by the SDK background task use to monitor the connection to Micro-RDK.
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:67]:
```suggestion
In some cases your connection to the ESP32 with an SDK can fail, be unstable or be slow.
This is usually caused by the SDK background task that monitors the connection to the micro-RDK.
```

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 3]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
...
+We recommend the following changes to the default settings when connecting to an ESP32 if you run into similar issues.
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:68]:
```suggestion
We recommend the following changes to the default settings when connecting to an ESP32 if you run into similar issues:
```

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 4]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
...
+
+{{< tabs >}}
+{{% tab name="Python SDK" %}}
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:71]:
```suggestion
{{% tab name="Python" %}}
```
Usually we just label tabs python and go and SDK is implied

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 5]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
...
+            # Replace "<API-KEY>" (including brackets)
+            # with your machine's api key
+            api_key='<API-KEY>')
+    )
+    # Replace "<ROBOT-URL>" (including brackets) with your machine's url
+    return await RobotClient.at_address('<ROBOT-URL>', opts)
+```
+
+{{% /tab %}}
+{{% tab name="Golang SDK" %}}
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:99]:
```suggestion
{{% tab name="Go" %}}
```

**Review by sguequierre — COMMENTED** [review]
Thanks for adding this! LGTM with a couple changes but you'll need to lint with Prettier to pass the check. You can follow [these instructions for VS code](https://github.com/viamrobotics/docs?tab=readme-ov-file#lint-js-and-markdown-files-with-prettier-on-save) or just do `npm install --save-dev --save-exact prettier` and `npx prettier -w docs/get-started/installation/prepare/microcontrollers/_index.md` -- then commit the changes

**Comment by npentrel** [comment]
@npmenard also happy to do that for you if you want to hand that off here

**Comment by npmenard** [comment]
I just committed your suggestions, happy to let you take it form here and merge it in when you feel it's ready

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 6]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
...
+
+In some cases your connection to the ESP32 with an SDK can fail, be unstable or be slow.
+This is usually caused by the SDK background task that monitors the connection to the micro-RDK.
+We recommend the following changes to the default settings when connecting to an ESP32 if you run into similar issues:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python
+# Replace the connect function found in the cod sample tab with the following
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:75]:
```suggestion
# Replace the connect function found in the code sample tab with the following
```

File: docs/get-started/installation/prepare/microcontrollers/_index.md [hunk 7]
```diff
@@ -62,6 +62,71 @@ Configure your `esp32` board for your machine.
 {{% card link="/build/program/" %}}
 {{< /cards >}}
 
+## Recommendations when using an SDK
+
+In some cases your connection to the ESP32 with an SDK can fail, be unstable or be slow.
+This is usually caused by the SDK background task that monitors the connection to the micro-RDK.
+We recommend the following changes to the default settings when connecting to an ESP32 if you run into similar issues:
```
Comment (sguequierre) [inline docs/get-started/installation/prepare/microcontrollers/_index.md:69]:
```suggestion
We recommend the following changes to the default settings in your SDK code when connecting to an ESP32 if you run into similar issues:
```

**Pull request merged**